tools/lint/lint.py

```diff
@@ -433,6 +433,18 @@ def check_parsed(repo_root: Text, path: Text, f: IO[bytes]) -> List[rules.Error]
         if timeout_value != "long":
             errors.append(rules.InvalidTimeout.error(path, (timeout_value,)))
 
+    if source_file.content_is_ref_node or source_file.content_is_testharness:
```

Comment:
Looks this previously is source_file.testharness_nodes?

Reply:
Right, but looks like it's the same thing, and maybe safer and more direct:

https://github.com/web-platform-tests/wpt/blob/2562b459eb0189a79329e56a1530fa0972cb0533/tools/manifest/sourcefile.py#L645-L651

Looks like #40594 was not checked because it's not markup:

https://github.com/web-platform-tests/wpt/blob/2562b459eb0189a79329e56a1530fa0972cb0533/tools/lint/lint.py#L388-L389

Reply:
Thanks!